lxd/auth/oidc/oidc.go

```diff
@@ -169,30 +169,15 @@ func (o *Verifier) Login(w http.ResponseWriter, r *http.Request) {
 	handler(w, r)
 }
 
+// Logout deletes the ID and refresh token cookies and redirects the user to the login page.
 func (o *Verifier) Logout(w http.ResponseWriter, r *http.Request) {
-	// Access token.
-	accessCookie := http.Cookie{
-		Name:     "oidc_access",
-		Path:     "/",
-		Secure:   true,
-		HttpOnly: false,
-		SameSite: http.SameSiteStrictMode,
-		Expires:  time.Unix(0, 0),
-	}
-
-	http.SetCookie(w, &accessCookie)
-
-	// Refresh token.
-	refreshCookie := http.Cookie{
-		Name:     "oidc_refresh",
-		Path:     "/",
-		Secure:   true,
-		HttpOnly: false,
-		SameSite: http.SameSiteStrictMode,
-		Expires:  time.Unix(0, 0),
+	err := o.setCookies(w, "", "", true)
+	if err != nil {
+		_ = response.ErrorResponse(http.StatusInternalServerError, fmt.Errorf("Failed to delete login information: %w", err).Error()).Render(w)
+		return
 	}
 
-	http.SetCookie(w, &refreshCookie)
+	http.Redirect(w, r, fmt.Sprintf("https://%s/ui/login", r.Host), http.StatusFound)
```

Comment:
I was wondering if the specific login URL could be sourced from a variable or made into a constant for later reference/change?

Reply:
I'm not sure. `/ui/login` is only used in this one place but `/ui/` is used as a magic string in multiple places. This might be better addressed in a later PR if we decide to parameterise URLs.